src/lib/bcrypt/index.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable require-jsdoc */
+import bcrypt from 'bcrypt';
+
+class Bcrypt {
+  async hashPwd(plainTextPassword) {
+    return bcrypt.hash(plainTextPassword, 10);
+  }
+
+  async comparePwd(plainTextPassword, hash) {
+    return bcrypt.compare(plainTextPassword, hash);
+  }
+}
+
+export default new Bcrypt();
```

Comment:
Based on my previous comment, lets have this like below:

```suggestion
export default Bcrypt;
```